specification/electronic-prescription-service-api.yaml

```diff
@@ -312,18 +314,74 @@ paths:
                 Bundle:
                   value:
                     $ref: components/examples/errors/prescription-order-update/bundle-error.json
+  /FHIR/R4/Task:
+    post:
+      operationId: return / withdraw
+      summary: Send either a Task-dispense-notification-amend or Task-dispense-proposal-return to EPS
+      description: |
+        ## Overview
+        This endpoint can be used to return a downloaded prescription to Spine, so that another dispensing system can download it. 
+        It can also be used to withdraw a dispensed prescription to Spine to notify Spine that something has gone wrong with a prescription that has been dispensed. 
```

Comment:
It can also be used to withdraw a dispense notification message from Spine, i.e. mark a dispense notification in error